DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -327,5 +317,37 @@ public void Get_section_content_should_order_by_tutorial_id_when_shared_order_by
 
             result.Tutorials.Select(tutorial => tutorial.Id).Should().Equal(expectedTutorialOrder);
         }
+
+        [TestCase(46, 353, 50, 45633, 103)]
+        [TestCase(262288, 22400, 386, 392, 225371)]
+        [TestCase(1, 9850, 101, 170, 284965)]
+        [TestCase(254480, 24224, 101, 1, null)]
+        public void Get_section_content_should_have_same_tutorials_as_stored_procedure(
+            int candidateId,
+            int customisationId,
+            int centreId,
+            int sectionId,
+            int? progressId
+        )
+        {
+            using (new TransactionScope())
+            {
+                // Given
+                var validProgressId = progressId ?? courseContentTestHelper.CreateProgressId(customisationId, candidateId, centreId);
+
+                var tutorialIdsReturnedFromOldStoredProcedure = sectionContentTestHelper
+                    .TutorialsFromOldStoredProcedure(validProgressId, sectionId)
+                    .Select(tutorial => tutorial.TutorialID);
+
+                // When
+                var tutorialIdsInSectionContent = sectionContentService
+                    .GetSectionContent(customisationId, candidateId, sectionId)?
+                    .Tutorials
+                    .Select(tutorial => tutorial.Id);
+
+                // Then
+                tutorialIdsInSectionContent?.Should().BeEquivalentTo(tutorialIdsReturnedFromOldStoredProcedure);
```

Comment:
Does this ignore order? If it does, can we change it so it will fail if the order is different? I think we did that for a different test? 🙂

Reply:
Ah yep you're right